frontend/playwright-tests/medicare_hiv.nightly.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test } from '@playwright/test'
+
+test('HIV conditions and medication adherence', async ({ page }) => {
```

Comment:
maybe add "PHRMA" or "Medicare" into this test title so it's easier to identify in the future